src/repositorios/injury.test.js

```diff
@@ -47,7 +49,8 @@ describe(
                                         id: 1,
                                         nome: expect.any(String),
                                         detalhes: expect.any(String),
-                                        grade: expect.any(Number)
+                                        grade: expect.any(Number),
+                                        id_collection: expect.any(Number)
```

Comment:
Test might fail if collection doesn't exist. Fix it using https://github.com/CRICDatabase/searchable-image-database-nodejs/issues/136.